package.json

```diff
@@ -178,9 +178,9 @@
     "stylelint-config-standard": "^34.0.0",
     "stylelint-processor-styled-components": "^1.10.0",
     "terser-webpack-plugin": "^5.3.10",
-    "tsx": "^4.7.0",
     "tsup": "^8.0.1",
-    "typescript": "^5.3.3",
+    "tsx": "^4.7.0",
+    "typescript": "5.4.0-dev.20240116",
```

Comment:
accidentally committed?

Reply:
No, just to test if all CI steps pass. I'm going to revert it.